### ISSUE TITLE
Show vertical line at that length - produced error

### ISSUE THREAD

**Issue by carrickpaul** [spyder-ide/spyder#17070]
### What steps will reproduce the problem?

uncheck "show vertical line at that length" checkbox, in "Completion and linting"

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\Paul\anaconda3\envs\algo_research\lib\site-packages\spyder\plugins\editor\plugin.py", line 412, in _rpc_call
    meth(*args, **kwargs)
  File "C:\Users\Paul\anaconda3\envs\algo_research\lib\site-packages\spyder\plugins\editor\plugin.py", line 1610, in call_all_editorstacks
    method = getattr(editorstack, method)
TypeError: getattr(): attribute name must be string
```

## Versions

* Spyder version: 5.1.5 
* Python version: 3.8.12
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.27.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.1.0 (OK)
nbconvert >=4.0               :  6.1.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso >=0.7.0;<0.9.0          :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0;<2.10.0        :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.4 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.2 (OK)
qtconsole >=5.1.0             :  5.1.1 (OK)
qtpy >=1.5.0                  :  1.10.0 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  58.0.4 (OK)
sphinx >=0.6.6                :  4.2.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.3 (OK)
zmq >=17                      :  22.2.1 (OK)

# Optional:
cython >=0.21                 :  None (OK)
matplotlib >=2.0.0            :  3.4.3 (OK)
numpy >=1.7                   :  1.21.2 (OK)
pandas >=1.1.1                :  1.3.3 (OK)
scipy >=0.17.0                :  1.7.1 (OK)
sympy >=0.7.3                 :  None (OK)
```

**Comment by dalthviz** [comment]
Hi @carrickpaul thank you for your feedback! This seems like a duplicate of https://github.com/spyder-ide/spyder/issues/16439 (which was fixed in Spyder 5.2.1). Could you update and try again? Thanks!

Note: If you are using the default conda channel to use Spyder currently I would suggest to create a new env to get the up-to-date spyder version using the conda-forge channel. For that you can use something like this:

    conda create -n spyder_env -c conda-forge python=3.8.12 spyder=5.2.1

Let us know if that helps!

**Comment by carrickpaul** [comment]
Thank you @dalthviz. Updating to 5.2.1 fixed it.
